scripts/00300-Aus.js

```diff
@@ -0,0 +1,7 @@
+const fullName = "Tangban Austin Bisong";
+const hngId = "HNG-00300";
+const language = "javascript";
+
+console.log(
+	`Hello World, this is ${fullName} with HNGi7 id ${hngId}, using ${language} for stage 2 task.`
```

Comment:
Please include your email address after the work `task` and separated by a space. As highlighted [here](https://hngi7.slack.com/archives/C014G2TMQ3G/p1591185461011700)